l10n_fi_banks/__init__.py

```diff
@@ -0,0 +1 @@
+import models
```

Comment:
At least one of the checks fail because of the `import models` still exists in the root `__init__.py`